docs/marketplace/tutorials/fully-functioning-rag-application.md

```diff
@@ -0,0 +1,172 @@
+
+After having configured the config map, we can save the configuration and move to the deploy.
+
+## 3. Deploy the configuration and generate embeddings
+
+From the _deploy_ section, we can deploy the new configuration. After verifying that the services are up, we can check if the application is running by accessing the chatbot frontend. The frontend is accessible through the automatically generated endpoint `/` (e.g., if the project is hosted at `https://my-project.console.my-company.com`, the frontend will be accessible at the same URL).
+
+![Assistant Playground homepage](../img/assistant-playground-home.png)
+
+The frontend will display a page with an input field where we can communicate with the chatbot. However, at this point, we don't have any embeddings, so specific questions may not give us the expected response.
+
+To see the list of all the APIs exposed by the configured services, we can go to the swagger UI of the _API Portal_. It can be accessed at the endpoint `/documentations/api-portal`. In the list of APIs, we can find two APIs with the tag _Embeddings_: `POST api/embeddings/generate` and `GET api/embeddings/status`.
+
+The `POST api/embeddings/generate` API allows us to generate embeddings starting from a webpage. The service will download the page, extract all the text, and generate embeddings from it. It will also search for links on the page and recursively generate embeddings from the linked pages that have the same domain and follow the specified filter path.
+
+Generating embeddings is an asynchronous task, meaning that the response is returned immediately, but the generation process continues in the background. To check the status of the generation process, we can use the `GET api/embeddings/status` API:
+
+- If the response is `{"status": "running"}`, it means the process is still ongoing.
+- If the response is `{"status": "idle"}`, it means there are no active processes at the moment (indicating that the previous process has finished).
+
+With this information, we can start generating our embeddings. We can make a request to the `api/embeddings/generate` API by expanding the corresponding card and clicking on "Try it out".
+
+![Embeddings generate API](../img/embeddings-generate-api.png)
+
+In the request body, we need to provide the following information:
+
+- `url` (mandatory): The starting page from which we want to generate embeddings.
+- `filterPath` (optional): A more specific path that will be used as a filter when finding new pages to download and analyze.
+
+Let's try with part of the Mia-Platform documentation:
+
+```json
+{
+    "url": "https://docs.mia-platform.eu/docs/fast_data/what_is_fast_data",
+    "filterPath": "https://docs.mia-platform.eu/docs/fast-data"
```

Comment:
this link is broken. could we use another example from our docs?

Reply:
That's the best we can do.

```suggestion
    "filterPath": "https://docs.mia-platform.eu/docs/fast_data"
```